docs/learn/learn-cross-consensus.md

```diff
@@ -4,7 +4,7 @@ title: Cross-Consensus Message Format (XCM)
 sidebar_label: Cross-Consensus Message Format (XCM)
 description: Learn about the messaging format at the forefront of interoperability.
 keywords: [cross-consensus, XCM, XCMP, interoperability, communication]
-slug: ../learn-crosschain
+slug: ../learn-xcm
```

Comment:
There are articles in circulation with the Polkadot Wiki links. Just ensure that they are not broke with appropriate redirects, if you are changing the slugs.

Reply:
I've updated in all places in the wiki. I think even though it might break some links out there in the wild, the rename is necessary.

Reply:
The suggestion was to implement redirects. Not to stop appropriate renaming the link.
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects

We do not know how many websites have linked our Wiki article, and as a rule of thumb, we need to ensure every linked Wiki doc is accessible.

<img width="1395" alt="image" src="https://user-images.githubusercontent.com/86818441/180180374-fcbdc6f2-d541-40d9-870d-9ba1715ad3e9.png">